examples/airgap/main.tf

```diff
@@ -0,0 +1,98 @@
+locals {
+  vpc_id         = "vpc-aabbccddeeffgghhiijj"
+  tag            = "tfe"
+  setup_bucket   = "mycompany-tfe-setup"
+  domain         = "mycompany-internal.net"
+  hostname       = "tfe"
+  license_file   = "mylicense.rli"
+  airgap_package = "v201911-1.airgap"
+  region         = "us-west-2"
+}
+
+provider "aws" {
+  region = "${local.region}"
+}
+
+module "external" {
+  source  = "hashicorp/terraform-enterprise/aws/modules/external-services"
+  version = ">= 0.0.5-beta"
```

Comment:
do we want to configure it not pinned to a specific version? if they reinit the configs after we release a higher version, they'll pull down that one whether they intend to or not?

Reply:
I had the same question, I'm unsure what we want to advocate for.

Reply:
I'd advocate for pinning the version so a customer is explicit when they perform infrastructure upgrades (like if they intend to put this in an existing CI pipeline)

Reply:
Yup, did that.